contracts/libraries/Helpers.sol

```diff
@@ -58,6 +58,7 @@ library Helpers {
 
 library MessageSigning {
     function recoverAddressFromSignedMessage(bytes signature, bytes message) internal pure returns (address) {
+        require(message.length == 116);
```

Comment:
for me, it is not logical to put the message length check here since, logically it is assumed that we can recover an address for the message of any length.